packages/story-editor/src/components/richText/karma/inlineSelection.karma.js

```diff
@@ -151,28 +174,34 @@ describe('CUJ: Creator can Add and Write Text: Select an individual word to edit
       expect(bold.checked).toBe(true);
       expect(italic.checked).toBe(true);
       expect(underline.checked).toBe(true);
-      // Note that entire selection is made black, because some part was black, when bold was pressedexpect(fontWeight.value).toBe('Black');
+      // Note that entire selection is made black, because some part was black, when bold was pressed
+      expect(fontWeight.value).toBe('Black');
       expect(letterSpacing.value).toBe('100%');
-      expect(fontColor.hex.value).toBe('00FF00');
+      expect(fontColor.hex.value).toBe('EEEEEE');
 
       // Exit edit-mode
-      await data.fixture.events.keyboard.press('Escape');
```

Comment:
For some reason, `Escape` did not properly exit edit mode in CI (locally everything worked), so had to use an alternative here for ensuring exiting edit mode.

Reply:
Hmm weird? Don't we use <kbd>escape</kbd> to exit edit-mode in other tests?

Reply:
We do, and it works generally, but for some reason in CI only (works locally) it doesn't seem to work in case of partial text selection 🤷